src/modules/shared/components/PhoneCard/PhoneCard.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import cn from 'classnames';
+import style from './PhoneCard.module.scss';
+
+export type PhoneItem = {
+  id: string;
+  name: string;
+  image: string;
+  price: number;
+  fullPrice: number;
+  year: number;
+  screen?: string;
+  capacity?: string;
+  ram?: string;
+  color?: string;
+  description: {
+    title: string;
+    text: string[];
+  }[];
+};
```

Comment:
You don't need to write everything, just what the component will accept because it should be reused everywhere. Also, please write it down through props

Reply:
I actually wanted to create separate file for this and just import it where we need it. I wrote more than just what I need, as later in PhonePage we would need more than we use in ProductCard